corehq/apps/geospatial/management/commands/index_geolocation_case_properties.py

```diff
@@ -15,3 +21,32 @@ def handle(self, *args, **options):
         case_type = options.get('case_type')
         query_limit = options.get('query_limit')
         chunk_size = options.get('chunk_size')
+
+
+def _case_query(domain, geo_case_property, case_type=None, size=None):
```

Comment:
nit: consider renaming to `_es_case_query` or something similar to indicate this is an es query.

Reply:
Sounds good, addressed in [807e5c8](https://github.com/dimagi/commcare-hq/pull/35090/commits/807e5c8c16605a3f625f5c82cc998d0588e55b1d).